mw-webapp/src/component/tooltip/Tooltip.module.scss

```diff
@@ -0,0 +1,41 @@
+.wrapper {
+  position: relative;
+}
+
+.target {
+  cursor: pointer;
+
+  &:hover + .tooltip {
+    display: block;
+  }
+}
+
+.tooltip {
+  display: none;
+  position: absolute;
+  overflow: hidden;
+
+  &_top {
+    bottom: 130%;
+    transform: translateX(-50%);
```

Comment:
could you explain how this two lines works? why do you need transform and left at the same time?

Reply:
``left: 50%`` - centers content in the center ``.wrapper``
``transform: translateX(-50%)`` - makes the content center and wrapper center coaxial